### ISSUE TITLE
Interactive Widgets

### ISSUE THREAD

**Issue by rt596** [MM2-0/Kvaesitso#274]
Widgets provide from other apps aren't interactive, so it would be great if one could interact with theme with additional widgets.

**Comment by MM2-0** [comment]
> Widgets provide from other apps aren't interactive

Except they are?

**Comment by rt596** [comment]
No they aren't! So i tried again with k-9mail. Can't scroll throung my email nor app opened when i click on widget. While Moon+Reader did opened ebook but couldn't scroll on the widget.

**Comment by MSBack** [comment]
Is Kvaesitso even compatible with widgets from other apps? I thought only the built in ones were available. It would be nice to be able to use widgets from other apps.

**Comment by MM2-0** [comment]
> Can't scroll throung my email nor app opened when i click on widget

Oh, you are right, scrolling is a bit akward, due to the scrolling being nested. You can try scrolling with two fingers, it works most of the time.

**Comment by kud** [comment]
Yes, I was also disturbed by this but now that I know the workaround, it's okish.

**Comment by MM2-0** [comment]
Fixed in upstream library
